pkg/controller/storagecluster/reconcile.go

```diff
@@ -43,6 +43,11 @@ type StorageClassProvisionerType string
 // ensureFunc which encapsulate all the 'ensure*' type functions
 type ensureFunc func(*ocsv1.StorageCluster, logr.Logger) error
 
+const (
+	dataPVCName     = "data"
+	metadataPVCName = "metadata"
```

Comment:
I think it is a good practice to keep all the constants in the same block for the same file :)

Reply:
Done and noted :)